auth-web/public/config/configuration.json

```diff
@@ -1,35 +1,36 @@
 {
+  "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
+  "AFFIDAVIT_FILE_SIZE": "73KB",
+  "API_DOCUMENTATION_URL": "https://developer.bcregistry.daxiom.ca/apis-summary/",
+  "AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca",
+  "AUTH_API_VERSION": "/api/v1",
+  "AUTH_LD_CLIENT_ID": "5db9da115f58e008123cd783",
   "AUTH_WEB_URL": "https://dev.bcregistry.ca/business/auth/",
-  "REGISTRY_HOME_URL": "https://dev.bcros.ca/",
+  "BCEID_OSD_LINK": "https://www.test.bceid.ca/os/?7686&SkipTo=Basic",
   "BUSINESS_CREATE_URL": "https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/",
+  "CORPORATE_ONLINE_URL": "https://dev.corponline.gov.bc.ca/",
   "DIRECTOR_SEARCH_URL": "https://dev.bcregistry.ca/directorsearch/",
-  "NRO_URL": "https://dev.bcregistrynames.gov.bc.ca/nro/",
-  "NAME_REQUEST_URL": "https://dev.bcregistry.ca/namerequest/",
   "ENTITY_SELECTOR_URL": "https://entity-selection-dev.apps.silver.devops.gov.bc.ca/",
-  "AUTH_LD_CLIENT_ID": "5db9da115f58e008123cd783",
-  "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
-  "BCEID_OSD_LINK": "https://www.test.bceid.ca/os/?7686&SkipTo=Basic",
+  "FAS_WEB_URL": "https://fas-dev.apps.silver.devops.gov.bc.ca/",
   "FILE_SERVER_URL": "https://minio-dev.apps.silver.devops.gov.bc.ca/public",
-  "AFFIDAVIT_FILE_SIZE": "73KB",
-  "PAYMENT_PAYEE_NAME": "BC Registries and Online Services",
-  "SENTRY_DSN": "",
-  "SENTRY_ENABLE": "false",
-  "VON_API_URL": "https://orgbook.gov.bc.ca",
-  "VON_API_VERSION": "/api/v3",
-  "AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca",
-  "AUTH_API_VERSION": "/api/v1",
-  "STATUS_API_URL": "https://status-api-dev.apps.silver.devops.gov.bc.ca",
-  "STATUS_API_VERSION": "/api/v1",
-  "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca",
-  "PAY_API_VERSION": "/api/v1",
+  "HOTJAR_ID": "",
```

Comment:
Empty for local development, otherwise it's configured in 1Password in the new "hotjar" vault.

@pwei1018 FYI ^^^